### PR TITLE
stop manually defining physical constants

### PR DESCRIPTION
**Pull request by dagar**
Requires https://github.com/PX4/ecl/pull/422

### CODE REVIEW

File: src/modules/mavlink/mavlink_receiver.cpp [hunk 1]
```diff
@@ -1922,9 +1920,9 @@ MavlinkReceiver::handle_message_hil_sensor(mavlink_message_t *msg)
 		struct accel_report accel = {};
 
 		accel.timestamp = timestamp;
-		accel.x_raw = imu.xacc / mg2ms2;
-		accel.y_raw = imu.yacc / mg2ms2;
-		accel.z_raw = imu.zacc / mg2ms2;
+		accel.x_raw = imu.xacc / CONSTANTS_ONE_G / 1000.0f;
```
Comment (bkueng) [inline src/modules/mavlink/mavlink_receiver.cpp]: This is wrong: brackets are missing

File: src/modules/simulator/simulator_mavlink.cpp [hunk 2]
```diff
@@ -1036,9 +1031,9 @@ int Simulator::publish_sensor_topics(mavlink_hil_sensor_t *imu)
 		struct accel_report accel = {};
 
 		accel.timestamp = timestamp;
-		accel.x_raw = imu->xacc / mg2ms2;
-		accel.y_raw = imu->yacc / mg2ms2;
-		accel.z_raw = imu->zacc / mg2ms2;
+		accel.x_raw = imu->xacc / CONSTANTS_ONE_G / 1000.0f;
```
Comment (bkueng) [inline src/modules/simulator/simulator_mavlink.cpp]: Same: missing brackets

**Comment by LorenzMeier** [comment]
I'm only just about convinced this is good cleanup because we're linking drivers with ECL and that dependency makes absolutely no sense. However, given the nuisance of having a potential major failure because different quantities of G were used I think its the better tradeoff.

**Comment by dagar** [comment]
Which part of that do you find most offensive?

My intention was to split the ECL project into separate libraries, at that point the drivers would only depend on geo. This would happen after https://github.com/PX4/Firmware/pull/9292 is merged because it allows us to actually control these dependencies.

Another option might be finding a new home (header) for the physical constants.

**Pull request merged**